src/access/profiling/payu_manager.py

```diff
@@ -27,6 +44,86 @@ def get_component_logs(self, path: Path) -> dict[str, ProfilingLog]:
             dict[str, ProfilingLog]: Dictionary mapping component names to their ProfilingLog instances.
         """
 
+    @property
+    def nruns(self) -> int:
+        """Returns the number of repetitions for the Payu experiments.
+
+        Returns:
+            int: Number of repetitions.
+        """
+        return self._nruns
+
+    @nruns.setter
+    def nruns(self, value: int) -> None:
+        """Sets the number of repetitions for the Payu experiments.
+
+        Args:
+            value (int): Number of repetitions.
+        """
+        if value < 1:
+            raise ValueError("Number of runs must be at least 1.")
+        self._nruns = value
+
+    @property
+    def startfrom_restart(self) -> str:
+        """Returns the restart option for the Payu experiments.
+
+        Returns:
+            str: Restart option.
+        """
+        return self._startfrom_restart
+
+    @startfrom_restart.setter
+    def startfrom_restart(self, value: str) -> None:
+        """Sets the restart option for the Payu experiments.
+
+        Args:
+            value (str): Restart option.
+        """
+        self._startfrom_restart = value
+
+    def generate_experiments(self, branches: list[str]) -> None:
+        """Generates Payu experiments for profiling data generation.
+
+        Args:
+            branches (list[str]): List of branches to generate experiments for.
+        """
+
+        for branch in branches:
+            if branch in self.experiments:
+                logger.info(f"Experiment for branch {branch} already exists. Skipping addition.")
+            else:
+                self.experiments[branch] = ProfilingExperiment(self.work_dir / branch / self.config_name)
+
+    def run_experiments(self) -> None:
+        """Runs Payu experiments for profiling data generation."""
+
+        runner_config = {
+            "test_path": self.work_dir,
+            "repository_directory": self.config_name,
+            "running_branches": [],
+            "keep_uuid": True,
+            "nruns": [],
+            "startfrom_restart": [],
+        }
+
+        for path, exp in self.experiments.items():
+            print(path, exp.status)
+            if exp.status == ProfilingExperimentStatus.NEW:
+                runner_config["running_branches"].append(path)
+                runner_config["nruns"].append(self.nruns)
+                runner_config["startfrom_restart"].append(self.startfrom_restart)
+                exp.status = ProfilingExperimentStatus.RUNNING
```

Comment:
Nothing that this status is also misleading - at this line the status is queued to be queued 😁; once experiment-runner runs, then the experiments are "queued" 😂